docs/model_developers/modules/handler.rst

```diff
@@ -83,3 +93,82 @@ or update needs to be invoked.
 
 The context (See :class:`~inmanta.agent.handler.HandlerContext`) passed to most methods is used to
 report results, changes and logs to the handler and the server.
+
+Built-in Handler utilities
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The :term:`Inmanta Agent<agent>`, responsible for executing handlers has built-in utilities to help
+handler development. This section describes the most important ones.
+
+Logging
+"""""""
+
+The agent has a built-in logging facility, similar to the standard python logger. All logs written
+to this logger will be sent to the server and are available via the dashboard and the API.
```

Comment:
And in files:
- agent log on the same machine
- resource action logfile on the server

Reply:
added